### ISSUE TITLE
1.1.24/1.1.26 unable to import amplitude/business central schema (was able to do this previously) Error wile retrieving existing schema. status code: 404; 

### ISSUE THREAD

**Issue by ann8ty** [fivetran/terraform-provider-fivetran#327]
**Describe the bug**
When schema breaks, one approach is to destroy the backend TF and re import the working schemas.  This isn't working today with the amplitude connector.  It worked previously on 1.1.24.  It does not work today on 1.1.24 or 1.1.26.

**To Reproduce**
```
data "aws_ssm_parameter" "ssm_amplitude_connector_id" {
  name = "/${local.app_env}/${local.schema_name}/amplitudeConnectorId"
}

# AWS_PROFILE="dbf" terraform import -var-file="./$ENVIROMENT_LONG_NAME/$CDK_PREFIX.tfvars"  fivetran_connector_schema_config.amplitude_schema cleansing_mandatory
resource "fivetran_connector_schema_config" "amplitude_schema" {
  connector_id           = data.aws_ssm_parameter.ssm_amplitude_connector_id.value
  schema_change_handling = local.default_schema_change_handling_is_allow_all
  schemas = {
    "${local.amplitude_connector_name}" = {
      enabled = true
    }
  }
}
```

**Expected behavior**
expected import to succeed

**Logs & Output**
```

(.venv) ~/code/data/data-infra/tf-fivetran-schemas $ AWS_PROFILE="dbf" terraform init -backend-config="./$ENVIROMENT_LONG_NAME/$CDK_PREFIX.tf" -reconfigure -upgrade


Initializing the backend...

Successfully configured the backend "s3"! Terraform will automatically
use this backend unless the backend configuration changes.

Initializing provider plugins...
- Finding hashicorp/aws versions matching "~> 5.32.1"...
- Finding hashicorp/external versions matching "~> 2.3.1"...
- Finding fivetran/fivetran versions matching "1.1.24"...
- Finding hashicorp/local versions matching "~> 2.4.0"...
- Using previously-installed hashicorp/local v2.4.1
- Using previously-installed hashicorp/aws v5.32.1
- Using previously-installed hashicorp/external v2.3.3
- Installing fivetran/fivetran v1.1.24...
- Installed fivetran/fivetran v1.1.24 (signed by a HashiCorp partner, key ID BD40EF4531102765)

Partner and community providers are signed by their developers.
If you'd like to know more about provider signing, you can read about it here:
https://www.terraform.io/docs/cli/plugins/signing.html

Terraform has made some changes to the provider dependency selections recorded
in the .terraform.lock.hcl file. Review those changes and commit them to your
version control system if they represent changes you intended to make.

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
(.venv) ~/code/data/data-infra/tf-fivetran-schemas $ AWS_PROFILE="dbf" terraform workspace select -or-create $CDK_PREFIX

AWS_PROFILE="dbf" terraform import -var-file="./$ENVIROMENT_LONG_NAME/$CDK_PREFIX.tfvars"  fivetran_connector_schema_config.amplitude_schema cleansing_mandatory 
data.aws_ssm_parameter.ssm_amplitude_connector_id: Reading...
data.aws_ssm_parameter.ssm_business_central_connector_id: Reading...
data.aws_ssm_parameter.fivetran_group_id_external_id: Reading...
data.aws_ssm_parameter.ssm_business_central_connector_id: Read complete after 0s [id=/ann5-datainfra-v2/schema-v2/businessCentralConnectorId]
data.aws_ssm_parameter.fivetran_group_id_external_id: Read complete after 0s [id=/ann5-datainfra-v2/schema-v2/fivetranGroupId]
data.aws_ssm_parameter.ssm_amplitude_connector_id: Read complete after 0s [id=/ann5-datainfra-v2/schema-v2/amplitudeConnectorId]
fivetran_connector_schema_config.amplitude_schema: Importing from ID "cleansing_mandatory"...
fivetran_connector_schema_config.amplitude_schema: Import prepared!
  Prepared fivetran_connector_schema_config for import
fivetran_connector_schema_config.amplitude_schema: Refreshing state... [id=cleansing_mandatory]
╷
│ Error: Unable to Read Connector Schema Resource.
│ 
│ Error wile retrieving existing schema. status code: 404; expected: 200; code: NotFound; message: HTTP 404 Not Found
╵

```

```
(.venv) ~/code/data/data-infra/tf-fivetran-schemas $ AWS_PROFILE="dbf" terraform import -var-file="./$ENVIROMENT_LONG_NAME/$CDK_PREFIX.tfvars"  fivetran_connector_schema_config.business_central_schema wearisome_perused
data.aws_ssm_parameter.ssm_business_central_connector_id: Reading...
data.aws_ssm_parameter.ssm_amplitude_connector_id: Reading...
data.aws_ssm_parameter.fivetran_group_id_external_id: Reading...
data.aws_ssm_parameter.ssm_amplitude_connector_id: Read complete after 0s [id=/ann5-datainfra-v2/schema-v2/amplitudeConnectorId]
data.aws_ssm_parameter.fivetran_group_id_external_id: Read complete after 0s [id=/ann5-datainfra-v2/schema-v2/fivetranGroupId]
data.aws_ssm_parameter.ssm_business_central_connector_id: Read complete after 0s [id=/ann5-datainfra-v2/schema-v2/businessCentralConnectorId]
fivetran_connector_schema_config.business_central_schema: Importing from ID "wearisome_perused"...
fivetran_connector_schema_config.business_central_schema: Import prepared!
  Prepared fivetran_connector_schema_config for import
fivetran_connector_schema_config.business_central_schema: Refreshing state... [id=wearisome_perused]
╷
│ Error: Unable to Read Connector Schema Resource.
│ 
│ Error wile retrieving existing schema. status code: 404; expected: 200; code: NotFound; message: HTTP 404 Not Found
╵

```

**Plugin version:**
1.1.24
1.1.26

**Additional context**

**Comment by beevital** [comment]
@ann8ty could you please share the real connector_id?

**Comment by beevital** [comment]
I can't investigate the case as the issue on API side. I need some anchor data to find logs.

**Comment by beevital** [comment]
>  re import the working schemas

You should not import schema. You can just re-apply settings over any state.
There's no any sense in importing.

**Comment by beevital** [comment]
The problem here is that for some reason connector doesn't have schema settings at all. In that case you just need to create a new resource without import. It will force terraform provider to reload connector schema from source and apply your settings.
Our public docs have appropriate statement:
https://registry.terraform.io/providers/fivetran/fivetran/latest/docs/resources/connector_schema_config#import

The use-case is invalid.

**Comment by beevital** [comment]
@ann8ty could you please clarify what do you mean by `When schema breaks` ?

**Comment by beevital** [comment]
Fixed in v1.2.0
Upstream changes on API side or source side are now handled softly by the provider on refresh.
In case if somethong went wrong with connector or source - you just need to refresh state. Or re-create a resource **without improting**.

**Comment by beevital** [comment]
Closing the issue as the fix was released. Feel free to re-open or open a new one id the fix didn't work for you.
